upnext/views.py

```diff
@@ -50,11 +50,14 @@ def login(request):
 def party_detail(request, party):
     get_object_or_404(Party, pk=party)
     party_obj = Party.objects.get(party_name = party)
+    tracks = Track.objects.filter(party_id=party_obj.party_name)
```

Comment:
Hmm, we shouldn't have to do this. A `Party` should know what `Tracks` belong to it, that's the logic behind a many-to-one relationship. Functionally equivalent, but I think using the relationship is cleaner. We can see if it works tomorrow! :)